src/components/pages/search.yml

```diff
@@ -16,3 +8,8 @@ search:
   lastUpdate:
     en: Last update
     de: Letzte Änderung
+  totalItems:
+    en: |
+      {count, plural, =1 {# result} other {# results}} found
+    de: |
+      {count, plural, =1 {# Ergebnis} other {# Ergebnisse}} gefunden
```

Comment:
I know it was like that before, but the pipe syntax should actually only be used for multiline texts (as it creates a line break after each line). It should be written like this:

~~~yml
  totalItems:
    en: "{count, plural, =1 {# result} other {# results}} found"
    de: "{count, plural, =1 {# Ergebnis} other {# Ergebnisse}} gefunden"
~~~

Reply:
Indeed, thanks for pointing this out. I've corrected this and all such misuse cases.